Homework_10/improved_calculator.py

```diff
@@ -0,0 +1,37 @@
+def addition(number_1, number_2, operataion):
+    if operataion == '+':
```

Comment:
I am not sure that we need if here. Or need to add else. Because if the operation is not + how you understand why your code is not working

Reply:
fixed